spec/spec_helper_acceptance.rb

```diff
@@ -1,8 +1,8 @@
 require 'voxpupuli/acceptance/spec_helper_acceptance'
 
 configure_beaker do |host|
-  install_module_from_forge_on(host, 'puppetlabs/apache', '>= 0')
-  install_module_from_forge_on(host, 'puppetlabs/java', '>= 0')
+  install_module_from_forge_on(host, 'puppetlabs/apache', '>= 0 < 7.0.0')
+  install_module_from_forge_on(host, 'puppetlabs/java', '>= 0 < 8.0.0')
```

Comment:
Why was it needed to limit these?

Reply:
Looks like the lack of upper bound caused trouble (no access to first commit logs but we see failures and no failure after adding these upper bounds).

I recently saw code without upper bounds I think, so maybe this was a regression that got fixed :shrug:.  Wana try to remove them?

Reply:
> Looks like the lack of upper bound caused trouble (no access to first commit logs but we see failures and no failure after adding these upper bounds).

I suspect the problem with this is that the module first installs all dependencies of this module. That includes stdlib 8. Then it tries to install `puppetlabs/java` which is incompatible with it. However, I'm not entirely sure.

Let's keep it for now and perhaps drop them if there is a new major release?